lib/plugins/print/print.js

```diff
@@ -91,9 +104,44 @@ class Print {
         this.serverless.variables.service = service;
         return this.serverless.variables.populateObject(service)
           .then((populated) => {
-            this.strip(populated);
-            const out = JSON.parse(jc.stringify(populated));
-            this.serverless.cli.consoleLog(YAML.dump(out, { noRefs: true }));
+            let conf = populated;
+            this.strip(conf);
+
+            // dig into the object
+            if (this.options.path) {
+              const steps = this.options.path.split('.');
+              for (const step of steps) {
+                conf = conf[step];
+              }
+            }
+
+            // apply an optional filter
+            if (this.options.filter) {
+              if (this.options.filter === 'keys') {
+                conf = Object.keys(conf);
+              } else {
+                throw new this.serverless.classes.Error('Filter must be "keys"');
+              }
+            }
+
+            // print configuration in the specified format
+            let out;
+
+            if (this.options.format === 'text') {
+              if (Array.isArray(conf)) {
```

Comment:
Please use `lodash` for consistency of other places since we have decided to use it for if-check.